var/spack/repos/builtin/packages/py-pybedtools/package.py

```diff
@@ -0,0 +1,43 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyPybedtools(PythonPackage):
+    """pybedtools wraps and extends BEDTools and offers
+    feature-level manipulations from within Python."""
+
+    homepage = "http://daler.github.io/pybedtools"
+    url      = "https://pypi.io/packages/source/p/pybedtools/pybedtools-0.7.10.tar.gz"
+
+    version('0.7.10', 'f003c67e22c48b77f070538368ece70c')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('bedtools2',     type=('build', 'run'))
+    depends_on('py-cython',     type=('build', 'run'))
```

Comment:
From `setup.py`:

> Cython is required to compile pybedtools from a development branch.

I don't think you should add the `cython` dependency since we are building a release version.